via/views/__init__.py

```diff
@@ -28,6 +28,10 @@ def add_routes(config):  # pragma: no cover
     config.add_route("proxy_d2l_pdf", "/d2l/proxied.pdf", factory=QueryURLResource)
     config.add_route("proxy_python_pdf", "proxied.pdf", factory=QueryURLResource)
 
+    config.add_route(
+        "api.youtube.transcript", "/api/youtube/transcript/{transcript_id}"
```

Comment:
Is this a new view?

Reply:
This is the new route for the new API endpoint, and the new view for it is in `views/api/youtube.py`